src/test/scala/com/cognite/spark/datasource/AssetsRelationTest.scala

```diff
@@ -10,22 +10,39 @@ class AssetsRelationTest extends FlatSpec with Matchers with SparkTest {
   val readApiKey = ApiKeyAuth(System.getenv("TEST_API_KEY_READ"))
   val writeApiKey = ApiKeyAuth(System.getenv("TEST_API_KEY_WRITE"))
 
+  val sourceDf = spark.read
+    .format("com.cognite.spark.datasource")
+    .option("apiKey", readApiKey.apiKey)
+    .option("type", "assets")
+    .load()
+  sourceDf.createTempView("sourceAssets")
+
+  val destinationDf = spark.read
+    .format("com.cognite.spark.datasource")
+    .option("apiKey", writeApiKey.apiKey)
+    .option("type", "assets")
+    .load()
+  destinationDf.createTempView("destinationAssets")
+
   it should "read assets" taggedAs ReadTest in {
-    val df = spark.read.format("com.cognite.spark.datasource")
+    val df = spark.read
```

Comment:
We haven't been running scalafmt on tests, which is why this commit has quite a few edits.

@wjoel Was there any reasoning for not running scalafmt on the test files? If not, I guess we should run it on all of them to clean up commits.

Reply:
Made a separate commit where i ran scalafmt on all test files. I can put it in a separate PR if you prefer that though.

Reply:
No reason. Separate PR would be the best, but we can also do it in a separate commit as part of this PR (if you're comfortable enough with git re-basing).